Resources/Prototypes/Entities/Structures/Holographic/projections.yml

```diff
@@ -45,17 +50,15 @@
         shape:
           !type:PhysShapeAabb
             bounds: "-0.5,-0.5,0.5,0.5"
-  #- type: TimedDespawn # DeltaV - no despawning for holofans
-  #  lifetime: 180
-  - type: Transform # DeltaV
-    noRot: true
+  - type: TimedDespawn
+    lifetime: 180
   - type: Airtight
     noAirWhenFullyAirBlocked: false
-  - type: Holofan # DeltaV - for whitelisting
-  - type: PointLight # DeltaV - lit holograms
-    enabled: true
-    radius: 3
-    color: yellow
+  # DeltaV Additions
+  - type: TimedDespawnDetailed
+    lifetime: 300
+  - type: Holofan
```

Comment:
Pretty sure this entity is still inheriting `- type: Holosign` from its parent as well.

I'm guessing that means:
- holofan projector can pick up only holofans
- but wet floor sign projector can pick up like, everything (wet floor signs, holofans, etc.)

Not sure if that's a major problem. I mean maybe a janitor can grief a bit by removing a safety-critical holofan that an atmos tech put down...

Reply:
If you wanted to fix this, simplest way would probably be to turn the HolosignWetFloor entity code above into just a BaseHoloprojection abstract entity *without* the Holosign component, and add a new little HolosignWetFloor entity child that adds the component